File: src/main/java/beans/Rsslink.java
package beans;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 * Tabella RSSLink
 */
@Entity
@Table(name = "`RSSLink`")
public class Rsslink {

	@Id
	@Column(name = "`id`")
	private long id;
	/**
	 * Lista link ultime news di BDO
	 */
	@Column(name = "`LastNewsBDO`")
	private String lastNewsBDO;
	@Column(name = "`LastPatchBDO`")
	/**
	 * Link ultima patch di BDO
	 */
	private String lastPatchBDO;
	/**
	 * Link ultimo feed di atm
	 */
	@Column(name = "`LastATMAlert`")
	private String lastAtmAlert;

	public Rsslink() {
	}


	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}


	public String getLastNewsBDO() {
		return lastNewsBDO;
	}

	public void setLastNewsBDO(String lastNewsBDO) {
		this.lastNewsBDO = lastNewsBDO;
	}


	public String getLastPatchBDO() {
		return lastPatchBDO;
	}

	public void setLastPatchBDO(String lastPatchBDO) {
		this.lastPatchBDO = lastPatchBDO;
	}


	public String getLastAtmAlert() {
		return lastAtmAlert;
	}

	public void setLastAtmAlert(String lastAtmAlert) {
		this.lastAtmAlert = lastAtmAlert;
	}

}

File: src/main/kotlin/it/discordbot/beans/RSSLink.kt
package it.discordbot.beans

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

/**
 * Entità che rappresenta la tabella contenitore dei link RSS
 * @property id Long id riga
 * @property lastNewsBDO String? link dell'ultima news di BDO
 * @property lastPatchBDO String? link dell'ultima patch di BDO
 * @property lastAtmAlert String? link dell'ultima news di ATM
 */
@Entity
@Table(name = "`RSSLink`")
class RSSLink {
	@Id
	@Column(name = "`id`")
	var id: Long = 0
	/**
	 * Lista link ultime news di BDO
	 */
	@Column(name = "`LastNewsBDO`")
	var lastNewsBDO: String? = null

	/**
	 * Link ultima patch di BDO
	 */
	@Column(name = "`LastPatchBDO`")
	var lastPatchBDO: String? = null
	/**
	 * Link ultimo feed di atm
	 */
	@Column(name = "`LastATMAlert`")
	var lastAtmAlert: String? = null
}